crates/derive/src/stages/attributes_queue.rs

```diff
@@ -143,8 +146,15 @@ where
             .prepare_payload_attributes(parent, batch.epoch())
             .await
             .map_err(StageError::AttributesBuild)?;
-        attributes.no_tx_pool = true;
-        attributes.transactions.extend(batch.transactions);
+        attributes.no_tx_pool = Some(true);
```

Comment:
Why is this an option in `OptimismPayloadAttributes`? Should always be there, doesn't have to be backwards compat w/ L1.